fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -167,6 +167,9 @@
     private OriginStatement origStmt;
     protected Map<String, String> sessionVariables = Maps.newHashMap();
 
+    private List<String> exportColumns ;
```

Comment:
If you store columns as a separate attribute in the export job from properties, you need to consider persistence.
Either reload the columns attribute during replay.
Or just persist the columns object directly.
My suggestion is not to modify the persistence logic. Re-parse columns after persistence.